### PR TITLE
WEBEU3 - Preprocessing-II - Evans Ibok

### PR DESCRIPTION
**Pull request by evansibok**

### CODE REVIEW

**Comment by evansibok** [comment]
Hello Toby! For some reason there are no node_modules files in the local directory of this project. So I'm left wondering why and when they show up. My less compiler works perfectly by the way.

File: index.html [hunk 1]
```diff
@@ -1,74 +1,109 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
```
Comment (OloruntobiAwoderu) [inline index.html:6]: You missed the viewport meta tag, and your media queries don't work due to that

File: less/home-page.less [hunk 2]
```diff
@@ -1,61 +1,126 @@
...
+			width: 30%;
+			margin-bottom: 30px;
+
+			@media (@mobile) {
+				width: 80%;
+			}
+
+			.btn {
+				.btn-font-property(1.5rem, 500, center);
+				.btn-body-property((100% - 30%), @button-bg, @white, 0.8em, 1em);
```
Comment (OloruntobiAwoderu) [inline less/home-page.less:118]: Nice use of Nesting!!!

File: less/variables.less [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 // Colors
-@white: #FFFFFF;
-@silver: #C0C0C0;
+@white: #ffffff;
+@silver: #c0c0c0;
 @shark: #212529;
-@eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@eastern-blue: #17a2b8;
```
Comment (OloruntobiAwoderu) [inline less/variables.less:12]: Nice use of variables

**Review by OloruntobiAwoderu — COMMENTED** [review]
Great Job overall on this project Evans!!! 
Keep up the good work

**Comment by OloruntobiAwoderu** [comment]
Also try and add the DS_store file created by your computer to the . gitignore file

File: index.html (hunk 1 above)
Comment (evansibok) [reply index.html:6]: Viewport meta tag added

File: less/home-page.less (hunk 2 above)
Comment (evansibok) [reply less/home-page.less:118]: Thank you!

**Pull request merged**